src/m365/teams/commands/cache/cache-remove.ts

```diff
@@ -71,34 +71,82 @@ class TeamsCacheRemoveCommand extends GraphCommand {
         logger.logToStderr('This command will execute the following steps.');
         logger.logToStderr('- Stop the Microsoft Teams client.');
         logger.logToStderr('- Clear the Microsoft Teams cached files.');
-  
+
         const result = await Cli.prompt<{ continue: boolean }>({
           type: 'confirm',
           name: 'continue',
           default: false,
           message: `Are you sure you want to clear your Microsoft Teams cache?`
         });
-        
+
         if (result.continue) {
           await this.clearTeamsCache(logger);
         }
-      }      
-    } 
+      }
+    }
     catch (err: any) {
       this.handleRejectedODataJsonPromise(err);
     }
   }
 
   private async clearTeamsCache(logger: Logger): Promise<void> {
     try {
-      await this.killRunningProcess(logger);
-      await this.removeCacheFiles(logger);
+      const folderExists = await this.checkIfCacheFolderExists(logger);
+      if (folderExists) {
+        await this.killRunningProcess(logger);
+        await this.removeCacheFiles(logger);
+        logger.logToStderr('Teams cache cleared!');
+      }
+      else {
+        logger.logToStderr('Cache folder does not exist. Nothing to remove.');
+      }
     }
     catch (e: any) {
       throw e.message as string;
     }
+  }
+
+  private async checkIfCacheFolderExists(logger: Logger): Promise<boolean> {
+    if (this.verbose) {
```

Comment:
Any specific reason why we're not using [fs.existsSync](https://nodejs.org/docs/latest-v18.x/api/fs.html#fsexistssyncpath) here which is a part of Node and already cross-platform?

Reply:
Hi @waldekmastykarz 

I was unaware of this functionality. I will change the code to use fs.existsSync.